src/index.html

```diff
@@ -2,13 +2,25 @@
 <html lang="en">
 <head>
 	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+
+	<title>Angular2 Seed</title>
 
 	<!-- inject:js -->
 	<!-- endinject -->
 
 	<!-- inject:css -->
 	<!-- endinject -->
+
+	<!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
+	<!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
+	<!--[if lt IE 9]>
+	<script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
+	<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
```

Comment:
`respond.js` is for IE 6-8 but Angular 2 only supports IE 9+ so I think it's safe to say we won't need it.